integration_test/run-adapter-and-wait.sh

```diff
@@ -1,8 +1,8 @@
 #!/bin/bash
-# apiStage project location dataset store adapter_port finish_port
-gradle run -Dexec.args='--dimse_aet=IMPORTADAPTER --dimse_port='$6' --dicomweb_addr=https://healthcare.googleapis.com/'$1'/projects/'$2'/locations/'$3'/datasets/'$4'/dicomStores/'$5'/dicomWeb --dicomweb_stow_path=/studies --verbose' &
+# apiStage project location dataset store adapter_port finish_port storescp_host store_scp_port
```

Comment:
Missing a comment for storage management host / port

Reply:
Ok, added